### PR TITLE
Add blog post on Grafana Monitoring for FerretDB

### PR DESCRIPTION
**Pull request by Fashander**
# Description

Closes FerretDB/engineering#77.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [ ] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by rumyantseva — COMMENTED** [review]
Looks good to me overall, left a few minor comments/quesitons

File: website/blog/2023-07-27-grafana-monitoring-ferretdb.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+authors: [alex]
+description: >
+  FerretDB is well suited to leverage the MongoDB data source plugin for Grafana to monitor and analyze your data.
+image: /img/blog/ferretdb-grafana.jpg
+tags: [compatible applications, tutorial]
+---
+
+![Grafana Monitoring for FerretDB](/img/blog/ferretdb-grafana.jpg)
+
+Monitoring tools play an essential role in overseeing the operations of a database, and [Grafana](https://grafana.com/) is one of the most popular monitoring tools.
```
[the remark this answers is not in the document]
Comment (rumyantseva) [reply website/blog/2023-07-27-grafana-monitoring-ferretdb.md:13]: P.S. Or maybe we do talk about monitoring... I'm not sure about the definition of monitoring in this particular case 🙈

File: website/blog/2023-07-27-grafana-monitoring-ferretdb.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+As the truly open-source alternative to MongoDB, [FerretDB](https://www.ferretdb.io/) helps you remain in the open source ecosystem and [avoid vendor lock-in](https://blog.ferretdb.io/5-ways-to-avoid-database-vendor-lock-in/).
+It also allows you to use the same commands and queries you're used to, along with many applications that MongoDB users already know.
+
+In this regard, FerretDB is well suited to leverage the [MongoDB data source plugin for Grafana](https://grafana.com/grafana/plugins/grafana-mongodb-datasource/).
+
+This blog post will cover Grafana and how you can use it to analyze and monitor your FerretDB data.
+
+## What is Grafana?
+
+Grafana is a comprehensive visualization and monitoring tool for managing and tracking data from multiple data sources such as MongoDB, ElasticSearch, PostgreSQL, MySQL, and Graphite, among others.
```
Comment (rumyantseva) [inline website/blog/2023-07-27-grafana-monitoring-ferretdb.md:26]: I'd also mention Prometheus as this feels like one of the main use cases :D But not critical.

**Review by rumyantseva — APPROVED** [review]
Looks good to me! I especially like that we show particular examples and explain how they work.

To me, the wording and everything else look clear, but one more review from either Alexey or Peter is needed (as usual).

File: website/blog/2023-07-27-grafana-monitoring-ferretdb.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+## What is FerretDB and How is it Different from MongoDB?
+
+FerretDB is a open source document database alternative to MongoDB that converts MongoDB wire protocol queries to SQL, with PostgreSQL and SQLite (Beta version) as the backend.
+
+By using FerretDB instead of MongoDB, you get the following benefits:
+
+- **Open source:** FerretDB is completely open source, released under the Apache 2.0 license – anyone can use it, modify it, and contribute to it.
+  On the other hand, MongoDB is no longer open source after changing their license to Server side Public License (SSPL), which leaves room for different interpretation.
+  For example, you may need to make your source code available if you provide MongoDB as a service.
```
Comment (ptrfarkas) [inline website/blog/2023-07-27-grafana-monitoring-ferretdb.md:44]: "For example, you may need to make your source code available if you provide MongoDB as a service. Also, the license does not allow you to embed MongoDB into your application."

**Review by ptrfarkas — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**